zipline/data/us_equity_pricing.py

```diff
@@ -662,7 +662,7 @@ def calc_dividend_ratios(self, dividends):
                     ratio = 1.0 - amount / prev_close
                     ratios[i] = ratio
                     # only assign effective_date when data is found
-                    effective_dates[i] = div_adj_date.value
+                    effective_dates[i] = ex_date
```

Comment:
Ah.  Can we rename it to something like `prev_close_date`, since we're no longer using it as the adjustment date?  Otherwise this looks good to me.